### PR TITLE
Post/hackathon

### PR DESCRIPTION
**Pull request by agrabeli**
This post promotes OONI's upcoming hackathon in Berlin. 

Perhaps I've said "Join us" too many times in the post...? :p

### CODE REVIEW

**Comment by bassosimone** [comment]
I like it and I think it can be posted as is!

File: content/post/berlin-hackathon-2016.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: "OONI Hackathon: Join us to explore internet censorship!"
+author: "Maria Xynou"
+date: "2016-10-18"
+tags: ["hackathon"]
+categories: ["report"]
```
Comment (agrabeli) [inline content/post/berlin-hackathon-2016.md]: @hellais You're right, I'll delete it.

File: content/post/berlin-hackathon-2016.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+date: "2016-10-25"
+tags: ["hackathon"]
+categories: []
+---
+
+Interested in exploring (and exposing) internet censorship? Join the [Open Observatory of Network Interference (OONI)](https://ooni.torproject.org/) hackathon in Berlin!
+
+* **When?** Friday, **11th November** & Saturday, **12th November** 2016 
+
+* **Where?** The **OnionSpace** (Gottschedstraße 4, 13357 Berlin, Germany)
```
Comment (darkk) [inline content/post/berlin-hackathon-2016.md]: What about making the name or the address a xref to OSM: http://www.openstreetmap.org/?mlat=52.55036&mlon=13.36964#map=18/52.55036/13.36964 ?
I don't know exact POI, so the marker may be misaligned.

File: content/post/berlin-hackathon-2016.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+![OONI Explorer](/post/berlin-hackathon/explorer.png)
+
+But our work is far from done and we need your help! Below you can learn about
+*some* of the ways that you can contribute to the fight against internet
+censorship through this hackathon.   
+
+# Hack on censorship tests 
+
+OONI has multiple [free software tests](https://github.com/TheTorProject/ooni-probe/tree/master/ooni/nettests) that you can hack on! These tests are designed to examine the following:
```
Comment (anadahz) [inline content/post/berlin-hackathon-2016.md]: Perhaps change `free software`  to `network interference`?
Suggestion: add the URL of the test specification: https://github.com/TheTorProject/ooni-spec/tree/master/test-specs

File: content/post/berlin-hackathon-2016.md (hunk 2 above)
Comment (anadahz) [inline content/post/berlin-hackathon-2016.md]: The (approximate) exact location (Ausgang 4) is here: https://www.openstreetmap.org/#map=19/52.54974/13.37013

**Comment by hellais** [comment]
Yes this looks good. I am going to merge.

**Pull request closed without merging**